deconz/build.yaml

```
@@ -7,4 +7,4 @@ codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
 args:
-  DECONZ_VERSION: 2.25.3
+  DECONZ_VERSION: 2.26.7
```

Here is a typo. It should be 2.27.6 I guess.
```suggestion
  DECONZ_VERSION: 2.27.6
```

---

Interesting, show how was this PR tested?

---

I committed from the wrong folder 🙈 
During local testing I copied the `deconz` addon folder to a local one to only have this addon for testing as I tried following [the documentation about local testing](https://developers.home-assistant.io/docs/add-ons/testing/). 
In order to commit the changes I should have synced the changes back but did them manually...

---

I just checked the version in my dev-installation and can confirm that I indeed tested the 2.27.6 version.

The docker image also shows deCONZ in the desired version:
```bash
docker run -it --entrypoint bash local/homeassistant/aarch64-addon-deconz:6.24.0
root@a07e9d4a3358:/# dpkg -l deconz
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================
ii  deconz         2.27.6       arm64        Zigbee monitoring and control.
```